tests/cluster/cluster.go

```diff
@@ -0,0 +1,181 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2018 Red Hat, Inc.
+ *
+ */
+
+package cluster
+
+import (
+	"fmt"
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	restclient "k8s.io/client-go/rest"
+	"github.com/pkg/errors"
+	"k8s.io/client-go/kubernetes"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	netclient "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/client/clientset/versioned/typed/k8s.cni.cncf.io/v1"
+	netv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+
+	"github.com/kubevirt/ovs-cni/tests/node"
+)
+
+var Kubeconfig *string
```

Comment:
I was sure we need it here but in fact we don't. moved to private inside test_suite.go.
Done

Reply:
In fact, we should try to avoid any globals, especially in a utils lib. Would you create a helper function initializing a needed client for each user?

Reply:
isn't keeping it private global in `tests_suite_test.go` safer?

Reply:
No, utils are imported by  the test suite, they must not depend on a global kept in it.

For utils, clients should be initialized inside helper functions OR passed down as an attribute, but that makes it unnecessarily verbose.

For tests that need a client, you can use a global initialized in the `BeforeSuite`